.editorconfig

```diff
@@ -1,5 +1,16 @@
 root = true
 
-[*.js]
-indent_style=tab
-trim_trailing_whitespace=true
\ No newline at end of file
+[*]
+indent_style = tab
+indent_size = 2
```

Comment:
My question was: What is the effect of `indent_size` when using `indent_style = tab`? Afaik it's only effect is how width the editor displays tabs. So it's only a visual effect. If this is true the next question is: Can we omit `indent_size` in `.editorconfig` so the editor (default) setting comes into effect? Why: Because every collaborator then sees his own default settings.